components/Dashboard/candidate/profile/update-skills-modal.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useState,KeyboardEvent } from 'react';
+import Modal from '@/components/modal/Modal';
+import { CancelIcon } from '@/assets';
+import { Button, OutlineBtn } from '@/components/button';
+interface modalcompProps {
+    open: boolean;
+    setOpen: any;
+    onClick: any;
+    loading: boolean;
+    setData: any;
+    data: any;
+}
+
+
+
+
+
+const UpdateSkillModal : React.FC<modalcompProps> = ({ open, setOpen, onClick, setData, loading,data }) => {
+    const [keyword, setKeyword] = useState("");
+    const [list, setList] = useState<any>([]);
```

Comment:
Type the list properly and not _any_
Default specifies it's an array, so the type inference should <Array<OfTheListType>>

```suggestion
    const [list, setList] = useState<Arrary<OfTheListType>>([]);
```